src/quantum/config.js

```diff
@@ -208,7 +208,7 @@ const PLATFORMS = [
   {
     browser: 'geckoview',
     label: 'Geckoview p2 aarch64',
-    platform: 'geckoview-p2-aarch64',
+    platform: 'p2-aarch64',
```

Comment:
@arnold-iakab @airimovici  and here we see clarifying the distinction between `platform`  and `browser`.  It should have been done a long time ago, but it was expedient at the time.  Now we need clarity so our reference data can match the `platform` of sample `platform`